code/game/turfs/closed/minerals.dm

```diff
@@ -54,6 +54,9 @@
 		return
 
 	var/mob/living/bumping = bumped_atom
+	if(bumping.combat_mode) // We don't allow combat mode mining (note this prevents fauna from mining on their own. Yes this is very stupid)
```

Comment:
the old version used isadvancedtooluser instead. i think thats a better check and i dont know why we want combat mode to be the limiter

Reply:
I agree, also I'm pretty sure everyone has combat mode. Pressing F will toggle even if you can't see the UI, you know because you can hear the sound.

Reply:
wait a moment
what the hell is going on here. why is this the only thing that prevents the mining?

Reply:
WHY DO MINERAL TURFS HAVE A CHECK THAT LETS SIMPLE MOBS KNOCK WALLS DOWN
WHY DOES NOTHING USE THIS
WHAT THE HELL